lib/mumukit/isolated_environment.rb

```diff
@@ -13,7 +13,8 @@ def configure!(*files)
       binds = dirnames.map { |it| "#{it}:#{it}" }
       volumes = Hash[dirnames.map { |it| [[it, {}]] }]
 
-      command = yield(*filenames).split
+      command = yield(*filenames)
+      command = command.split if String === command
```

Comment:
Please use `is_a?`, let's avoid `===` unless we really need a polymorphic behavior